### PR TITLE
SMS inbound should be logged in django

### PR DESCRIPTION
**Pull request by gsvr**
https://praekelt.atlassian.net/browse/RZ-45

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: js_sandbox/lib/go-rts-zambia-sms.js [hunk 1]
```diff
@@ -183,20 +183,51 @@ function GoRtsZambiaSms() {
...
+                                p_sms.add_callback(function(result) {
+                                    var url = "data/headteacher/" + '?' + self.url_encode({
+                                        'zone': zone_id,
+                                        'is_zonal_head': 'true'
+                                    });
+                                    // get the zonal head
+                                    return self.cms_get(url);
+                                });
+                                p_sms.add_callback(function(result) {
+                                    var zonalhead = result.objects[0];  // assumes only one zonal head
```
Comment (imsickofmaps) [inline js_sandbox/lib/go-rts-zambia-sms.js:218]: We need a fixture and test that returns an empty zonal head to be sure of this working

**Comment by imsickofmaps** [comment]
:+1: once we confirm the no-zonal-head thing
